src/lib/config.js

```diff
@@ -26,6 +26,10 @@ var getKubernetesROServiceAddress = function() {
 module.exports = {
   loopSleepSeconds: process.env.MONGO_SIDECAR_SLEEP_SECONDS || 5,
   unhealthySeconds: process.env.MONGO_SIDECAR_UNHEALTHY_SECONDS || 15,
+  mongoListenPort: process.env.MONGO_LISTEN_PORT || 27017,
+  mongoSSLEnabled: ( process.env.MONGO_SSL_ENABLED === 'true' ) || false,
+  mongoSSLAllowInvalidCertificates: ( process.env.MONGO_SSL_ALLOW_INVALID_CERTIFICATES === 'true' ) || true,
+  mongoSSLAllowInvalidHostnames: ( process.env.MONGO_SSL_ALLOW_INVALID_HOSTNAMES === 'true' ) || true,
```

Comment:
Also, once you pull, if you have a look at how `isConfigRS` is set for boolean values, use the same approach for these values. Extract it out into a common func too. Cheers.